misc/rott.desktop.in

```diff
@@ -2,8 +2,8 @@
 Type=Application
 Name=Rise of the Triad
 Comment=A high quality, fast scrolling first-person perspective 3D action game
-Icon=rott
-TryExec=rott
-Exec=rott
+Icon=@TARADINO_EXEC@
```

Comment:
Could you add the following changes, please:
- Rename the file to taradino.desktop
- `Name=Taradino`
- `Comment=SDL2 port of Rise of the Triad`
- `Keywords=first;person;shooter;fps;3d;rott;rise;of;the;triad;`

Reply:
Could you also rename all the icons in the `misc/` directory, adjust the `misc/icons.mk` Makefile and remove the useless `runrott.sh` script?